packages/expect/package.json

```diff
@@ -39,6 +39,7 @@
     "chai": "^4.3.7"
   },
   "devDependencies": {
+    "@vitest/expect": "workspace:*",
```

Comment:
Why would `expect` have itself in dependencies?

Reply:
typo 😅